sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -353,10 +356,30 @@ object DottyPlugin extends AutoPlugin {
           Def.valueStrict { scalaInstance.taskValue }
       }.value,
 
-      // We need more stuff on the classpath to run the `doc` task.
+      // Configuration for the doctool
+      resolvers ++= (if(!useScala3doc.value) Nil else Seq(
+        Resolver.jcenterRepo,
+        Resolver.bintrayRepo("kotlin", "kotlin-dev"),
+        Resolver.bintrayRepo("virtuslab", "dokka"),
+      )),
+      useScala3doc := false,
+      scala3docOptions := Nil,
+      Compile / doc / scalacOptions := {
+        val s3dOpts = scala3docOptions.value.map("--+DOC+" + _)
```

Comment:
What is this for? Some documentation would be useful

Reply:
I've added a comment here and created a issue for proper documentation: https://github.com/lampepfl/scala3doc/issues/222

Reply:
So the reason we need this is that we have a separate CLI interface with its own arguments, and we may need to forward options to it (that's what `scala3docOptions` keeps track of). When we receive options from SBT, we strip off the ones starting with `--+DOC+`, and pass these along with some default values we extracted from compiler args to the doctool arg parser.